### PR TITLE
[new release] tezos-context-hash and tezos-context-hash-irmin (1.0.0)

### PR DESCRIPTION
**Pull request by samoht**
Specification of the Tezos context hash

- Project page: <a href="https://github.com/tarides/tezos-context-hash">https://github.com/tarides/tezos-context-hash</a>

##### CHANGES:

- Initial release compatible with irmin.2.7

### CODE REVIEW

File: packages/tezos-context-hash-irmin/tezos-context-hash-irmin.1.0.0/opam [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  "tezos-context-hash"
+  "irmin" {>= "2.7.0"}
+  "cmdliner"
+  "fmt"
+  "yojson"
+  "irmin-pack"
+  "ppx_irmin"
+]
+build: [
+  ["dune" "subst"] {pinned}
```
Comment (samoht) [inline packages/tezos-context-hash-irmin/tezos-context-hash-irmin.1.0.0/opam:24]:
```suggestion
  ["dune" "subst"] {dev}
```

File: packages/tezos-context-hash/tezos-context-hash.1.0.0/opam [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+depends: [
+  "dune" {>= "2.6"}
+  "alcotest" {with-test}
+  "digestif" {>= "0.9.0"}
+  "fmt"
+  "repr"
+  "zarith"
+]
+build: [
+  ["dune" "subst"] {pinned}
```
Comment (samoht) [inline packages/tezos-context-hash/tezos-context-hash.1.0.0/opam:26]:
```suggestion
  ["dune" "subst"] {dev}
```

File: packages/tezos-context-hash/tezos-context-hash.1.0.0/opam [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  "Mattias Roux <mattias@tarides.com>"
+  "Thomas Gazagnaire <thomas@tarides.com>"
+]
+license: "MIT"
+homepage: "https://github.com/tarides/tezos-context-hash"
+bug-reports: "https://github.com/tarides/tezos-context-hash/issues"
+depends: [
+  "dune" {>= "2.6"}
+  "digestif" {>= "0.9.0"}
+  "fmt"
```
Comment (samoht) [inline packages/tezos-context-hash/tezos-context-hash.1.0.0/opam:20]:
```suggestion
  "fmt" {>= "0.8.7"}
```

File: packages/tezos-context-hash/tezos-context-hash.1.0.0/opam [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  "repr"
```
Comment (samoht) [inline packages/tezos-context-hash/tezos-context-hash.1.0.0/opam:21]:
```suggestion
  "repr" {>= "0.2.0"}
```

**Comment by samoht** [comment]
All green! 🎉

File: packages/tezos-context-hash-irmin/tezos-context-hash-irmin.1.0.0/opam [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+  "Clément Pascutto <clement@pascutto.fr>"
+  "Mattias Roux <mattias@tarides.com>"
+  "Thomas Gazagnaire <thomas@tarides.com>"
+]
+license: "MIT"
+homepage: "https://github.com/tarides/tezos-context-hash"
+bug-reports: "https://github.com/tarides/tezos-context-hash/issues"
+depends: [
+  "dune" {>= "2.6"}
+  "tezos-context-hash"
```
Comment (kit-ty-kate) [inline packages/tezos-context-hash-irmin/tezos-context-hash-irmin.1.0.0/opam:15]: is this going to be a package that you want to synchronize/release together with tezos-contex-hash-irmin or should they be more dislocated? (i.e. do you want `{= version}` or do you prefer to update previous constraints manually later?)

Comment (samoht) [reply packages/tezos-context-hash-irmin/tezos-context-hash-irmin.1.0.0/opam:15]: I think it's fine if they are released separately - tezos-context-hash doesn't have to follow irmin releases whereas tezos-context-hash-irmin has.

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**